fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/RootRdfContext.java

```diff
@@ -129,7 +132,25 @@ private void concatRepositoryTriples() {
                     createTypedLiteral(counters.get(PREFIX + FIXITY_REPAIRED_COUNTER).getCount()).asNode()));
         }
 
+        //add project build description triples here
+        final BuildPropertiesLoader buildLoader = new BuildPropertiesLoader();
+
+        final com.hp.hpl.jena.graph.Node releaseSubject = createURI(REPOSITORY_NAMESPACE + "release");
+            b.add(create(subject(), DOAP_RELEASE.asNode(), releaseSubject));
+            b.add(create(releaseSubject, type.asNode(), createURI(DOAP_NAMESPACE + "Version")));
```

Comment:
I mean that many open source projects contain a `doap.rdf` file (Description Of A Project). E.g. https://github.com/apache/storm/blob/master/doap_Storm.rdf, https://github.com/apache/kafka/blob/trunk/doap_Kafka.rdf. https://github.com/ruby-rdf/rdf/blob/develop/etc/doap.ttl

Since that file typically contains these sorts of values (major/minor version, etc), and since that file is always an RDF resource, it could be a convenient resource to link to from the repository.